src/common/components/widgets/modules/WidgetFrame.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+    <div class="widget-frame"
+         :class="{
+             [size]: true,
+         }"
+    >
+        <header>
+            <h3 class="title">
+                {{ title }}
+            </h3> <slot name="header-right" />
+        </header>
+        <div class="body">
+            <slot />
+        </div>
+        <footer>
+            <div class="footer-left">
+                <p-datetime-picker style-type="text" select-mode="range" :selected-dates.sync="selectedDates" />
+                <p-divider :vertical="true" />
+                <currency-select-dropdown />
+            </div>
+            <div class="footer-right">
+                <slot name="footer-right">
+                    <router-link v-if="widgetLink && !printMode" :to="widgetLink" class="anchor-button">
+                        {{ $t('BILLING.COST_MANAGEMENT.DASHBOARD.FULL_DATA') }}
+                        <p-i name="ic_arrow_right" width="1rem" height="1rem"
+                             color="inherit transparent"
+                        />
+                    </router-link>
+                </slot>
+            </div>
+        </footer>
+    </div>
+</template>
+
+<script lang="ts">
+import type { SetupContext } from 'vue';
+import { reactive, toRefs } from 'vue';
+
+import { PDatetimePicker, PDivider, PI } from '@spaceone/design-system';
+import { CARD_SIZE } from '@spaceone/design-system/src/data-display/cards/card/config';
+
+import { useProxyValue } from '@/common/composables/proxy-state';
+
+import CurrencySelectDropdown from '@/services/cost-explorer/modules/CurrencySelectDropdown.vue';
+
+export default {
+    name: 'WidgetFrame',
+    components: {
+        CurrencySelectDropdown,
+        PDatetimePicker,
+        PDivider,
+        PI,
+    },
+    props: {
+        title: {
+            type: String,
+            default: 'Title',
+        },
+        size: {
+            type: String,
+            default: CARD_SIZE.md,
+            validator(size: any) {
+                return Object.values(CARD_SIZE).includes(size);
+            },
+        },
+        widgetLink: {
+            type: [Object, String],
+            default: undefined,
+        },
+        noData: {
+            type: Boolean,
+            default: false,
+        },
+        printMode: {
+            type: Boolean,
+            default: false,
+        },
+        // currency
+    },
+    setup(props, { emit }:SetupContext) {
+        const state = reactive({
+            selectedDates: useProxyValue('selectedDates', props, emit),
+        });
+        return {
+            ...toRefs(state),
+        };
+    },
+};
+</script>
+
+<style lang="postcss" scoped>
+@define-mixin widget-size $widget-width {
+    & {
+        height: 29rem;
+        width: $widget-width;
+    }
+}
+
+.widget-frame {
+    border-color: theme('colors.gray.200');
+    display: inline-flex;
+    flex-direction: column;
+
+    @apply border rounded-lg bg-white;
```

Comment:
apply를 맨 위에 놓는 게 어떨까요?

```suggestion
.widget-frame {
    @apply border rounded-lg bg-white;
    border-color: theme('colors.gray.200');
    display: inline-flex;
    flex-direction: column;
```